code/matt_r/python/rot13.py

```diff
@@ -0,0 +1,20 @@
+
+
+
+
+pre_rot = input('Enter a word to rot.:  ')
```

Comment:
Doesn't handle capital letters or punctuation. Case could be handled using the string method .lower()